_posts/2024-07-22-kubeflow-1.9-release.md

```diff
@@ -0,0 +1,226 @@
+
+This initial release includes REST APIs and a Python SDK to track model artifacts and model metadata with a standardized format that can be reused across Kubeflow components, such as to deploy Inference Servers. You can get started by following the [Model Registry tutorial on the Kubeflow website](https://www.kubeflow.org/docs/components/model-registry/overview/), or see a short [demo video](https://www.youtube.com/watch?v=JVxUTkAKsMU) of the Model Registry in action.
+
+We are just getting started. This is an Alpha version and we look forward to feedback. The [model registry working group](https://docs.google.com/document/d/1DmMhcae081SItH19gSqBpFtPfbkr9dFhSMCgs-JKzNo/edit) meets biweekly: you can provide feedback by joining the meeting or directly on the [repository](https://github.com/kubeflow/model-registry/issues).
+
+## Fine-Tune APIs for LLMs
+
+In the rapidly evolving ML/AI landscape, the ability to fine-tune pre-trained models represents a significant leap towards achieving custom solutions with less effort and time. Fine-tuning with custom datasets allows practitioners to adapt large language models (LLMs) to their specific needs.
+
+However, fine-tuning tasks often require extensive manual intervention, including the configuration of training environments and the distribution of data across nodes. The new [Fine-Tune API](https://www.kubeflow.org/docs/components/training/user-guides/fine-tuning/) aims to simplify this process, offering an easy-to-use Python interface that abstracts away the complexity involved in setting up and executing fine-tuning tasks on distributed systems.
+
+By providing this API, Training Operator not only simplifies the user experience for ML practitioners but also leverages its existing infrastructure for distributed training. You can take advantage of Kubernetes’ ability to dynamically schedule GPU, thus saving on compute resources and cost. Training Operator also gives you fault tolerant guarantees, to save your training procedures from cluster node failures.
+
+## Pipelines
+
+### Argo Workflows and Tekton Backends Consolidation
+
+The Pipelines Tekton backend has been [merged](https://github.com/kubeflow/pipelines/pull/10678) into the main Kubeflow Pipelines repository. You can now choose what workflow engine to use from the same Pipelines version. This proves the extensibility and flexibility of the KFP v2 architecture, which encourages other contributors to bring support for other workflow engines.
+
+Compared to Argo Workflows, Tekton provides better extensibility thanks to pipeline definitions and reusable components that facilitate easier customization and integration into various CI/CD systems, making it adaptable for diverse machine learning workflows. Tekton also optimizes resource usage and execution efficiency, enhancing scalability for large-scale machine learning workloads compared to Argo Workflows, resulting in faster execution and better resource management.
```

Comment:
@StefanoFioravanzo Can we work on a more neutral description of the benefits of the kfp-tekton merge in KFP 2.2.0? We can focus more on having tekton added to KFPv2 because of its fexible and extensible architecture than comparing one or another.

Reply:
Rephrased section